src/python/library/tritonclient/grpc/_client.py

```diff
@@ -1502,6 +1522,7 @@ def wrapped_callback(call_future):
                 if request_id != "":
                     verbose_message = verbose_message + " '{}'".format(request_id)
                 print(verbose_message)
+            return self._call_future
```

Comment:
I think @nnshah1 had some concerns about returning this grpc future directly, not wanting users to call `future.result()` on it, possibly conflicting with our own callback that blocks on the result. However, I think it's fine to return it directly and document they shouldn't call `.result()` on it for now. 

I'll let Neelay comment on this.

Reply:
I would like us to define a "WrappedFuture" class, where we override the result() method to return an InferResult() (similar in that sense to the wrapped_callback).

I think returning a future but having the result() be different than with the callback will be non-intuitive. - 

Let me know if it's feasible.

Reply:
https://github.com/triton-inference-server/client/pull/393/commits/cdf8aea178dbadf7d40f247959d24915b8c4694f